### ISSUE TITLE
Minimum supported PyTorch version?

### ISSUE THREAD

**Issue by thomasjpfan** [data-apis/array-api-compat#24]
Similar to https://github.com/data-apis/array-api-compat/issues/21 but for other array libraries. For example, PyTorch `correction` was is in 2.0, but not in previous versions. The following fails for PyTorch < 2.0:

https://github.com/data-apis/array-api-compat/blob/e2203df4e412289ed64eeb70468dd02b0e72e607/array_api_compat/torch/_aliases.py#L378

For reference, here are the `torch.std` docs for [1.13](https://pytorch.org/docs/1.13/generated/torch.std.html?highlight=std#torch.std) and [2.0](https://pytorch.org/docs/2.0/generated/torch.std.html?highlight=std#torch.std).

**Comment by asmeurer** [comment]
`correction` isn't documented, but it is present in 1.13. 

```py
>>> import torch
>>> torch.__version__
'1.13.0'
>>> torch.std(torch.arange(3, dtype=torch.float32), (0,), correction=2)
tensor(1.4142)
```

I've tested all the pytorch support against 1.13 so things should work there. I haven't tested against 2.0 yet, but don't expect too many breakages. 

We should probably support at least 1.13 and 2.0 for now. Let me know if supporting anything earlier is important for you.

**Comment by rgommers** [comment]
`correction` was only recently updated to accept floats in addition to integers. Maybe that's the issue?

**Comment by thomasjpfan** [comment]
From the weekly call, `correction` must be an int to support 1.13.

**Comment by thomasjpfan** [comment]
As a minimum reproducer, the following fails on 1.13:

```python
import torch
import array_api_compat 

x = torch.asarray([1.0, 2.0, 3.0])
xp = array_api_compat.get_namespace(x)

xp.std(x)
# TypeError: std() received an invalid combination of arguments - got (Tensor, tuple, correction=float), but expected one of:
# * (Tensor input, tuple of ints dim, *, int correction, bool keepdim, Tensor out)
# * (Tensor input, tuple of ints dim, bool unbiased, bool keepdim, *, Tensor out)
# * (Tensor input, bool unbiased)
# * (Tensor input, tuple of names dim, *, int correction, bool keepdim, Tensor out)
# * (Tensor input, tuple of names dim, bool unbiased, bool keepdim, *, Tensor out)
```

Setting `correction` to an `int` works, i.e. `xp.std(x, correction=0)`.

**Comment by asmeurer** [comment]
The test suite xfail of this not working with nonintegral correction made me not realize this was broken with the default value. This is fixed in #25
